### PR TITLE
Add info about price additions

### PR DESCRIPTION
**Pull request by kamil-gwozdz**
vat_pct added to scale_prices

### CODE REVIEW

File: source/includes/v4/_events.md [hunk 1]
```diff
@@ -576,6 +576,7 @@ or scale prices with following attributes:
 - group (boolean), whether the price is for the whole group or per participant (per seat)
 - optional (boolean), whether the price is a choice or not
 - price_cents (integer) price in EUR cents
+- vat_pct (float), the pricing tax rate
```
Comment (douglasward) [inline source/includes/v4/_events.md:579]: @kamil98 can we update the example response as well?

**Comment by douglasward** [comment]
@kamil98 is the API version updated in go~mus as well?

**Comment by vollnhals** [comment]
You are both awesome. Thanks for doing this.

File: source/includes/v4/_events.md (hunk 1 above)
Comment (kamil-gwozdz) [reply source/includes/v4/_events.md:579]: do we even have an example response for scale prices?

**Comment by kamil-gwozdz** [comment]
@douglasward 
> @kamil98 is the API version updated in go~mus as well?

yes
https://giantmonkey.githost.io/gomus/gomus/merge_requests/3919

File: source/includes/v4/_events.md (hunk 1 above)
Comment (douglasward) [reply source/includes/v4/_events.md:579]: no, you are right.

**Pull request merged**